test/shared/src/main/scala/zio/test/mock/DebugProperties.scala

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2017-2020 John A. De Goes and the ZIO Contributors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package zio.test.mock
+
+/**
+ * Java properties debug output of ZIO Mock framework
+ *
+ * @see [[DebugProperties]]
+ */
+object DebugProperties {
+
+  /**
+   * To see mock debug output during test execution, set `-Dzio.test.mock.debug=true` system property!
+   *
+   * {{{
+   *   sbt -Dzio.test.mock.debug=true test
+   * }}}
+   */
+  final val `zio.test.mock.debug` = "zio.test.mock.debug"
```

Comment:
If possible, can we configure this through a TestAspect? We don't normally use system properties for anything in the library.

Reply:
It would require threading the TestAspect information to mock execution, not sure if possible without major refactoring.
It apprears System.properties fails to link on ScalaJS, so I switched to env variable instead.

This debug information is not really intended for the end user, but to developers when extending/debugging the Mocking framework itself.

Reply:
Is this supposed to be a user facing feature at all? Maybe it should just be a `private val` that contributors can set to `true` for debugging purposes when developing?

Reply:
Sure, that will work too. Updated code.